.github/workflows/CondaLock.yml

```diff
@@ -50,51 +50,66 @@ jobs:
            activate-environment: condalock
            use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
 
+      - name: Specify which architectures to build lockfiles for
+        run: |
+          # space delimited list of architectures to send to conda-lock
+          echo "BUILD_ARCHS='64 aarch64 ppc64le'" > ${GITHUB_ENV}
```

Comment:
i think it'd be best to just start with one additional? aarch64 and add others down the line if need be

Reply:
Let's build just for `linux-aarch64` first as mentioned at https://github.com/pangeo-data/pangeo-docker-images/issues/396#issuecomment-1643006035